variation/main.py

```diff
@@ -115,3 +117,27 @@ def normalize(q: str = Query(..., description=q_description),
             response_datetime=datetime.now()
         )
     )
+
+
+@app.get('/variation/translate_identifier',
+         summary='Given an identifier, use SeqRepo to return a list of aliases.',  # noqa: E501
+         response_description='A response to a validly-formed query.',
+         response_model=List[str],
+         description='Return list of aliases for an identifier'
+         )
+def translate_identifier(
+        identifier: str = Query(..., description='The identifier to find aliases for'),  # noqa: E501
+        target_namespaces: Optional[str] = Query(None, description='The namespaces of the aliases, separated by commas')  # noqa: E501
+) -> List[Optional[str]]:
```

Comment:
Should this just be `List[str]`?

Reply:
btw another way to get `target_namespaces` is by using a list parameter (https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#query-parameter-list-multiple-values). May or may not be more convenient.

Reply:
Yes